client/ayon_houdini/plugins/publish/validate_wait_for_render.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import hou
+
+import pyblish.api
+from ayon_core.pipeline import PublishValidationError
+from ayon_core.pipeline.publish import RepairAction
+
+from ayon_houdini.api import plugin
+
+
+class EnableWaitForRender(RepairAction):
+    label = "Enable WaitForRendertoComplete"
+    icon = "mdi.pencil-plus-outline"
+
+
+class ValidateWaitForRender(plugin.HoudiniInstancePlugin):
+    """Validate `WaitForRendertoComplete` is enabled.
+
+    Disabling `WaitForRendertoComplete` cause the local render to fail
+    as the publish execution continues while the render may not be finished yet.
+
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["usdrender"]
+    label = "Validate Wait For Render to Complete"
+    actions = [EnableWaitForRender]
```

Comment:
Any reason why we need to have the extra class inherited from the Repair action?
```suggestion
    actions = [RepairAction]
```

Reply:
> Any reason why we need to have the extra class inherited from the Repair action?

yes, so that I can change the label and the icon. 
Here's how the default repair action looks.
![image](https://github.com/user-attachments/assets/6748e419-e3fd-46d3-8686-28a2b3d177e6)

Here's how the extra class looks like. which makes the button action clearer.
![image](https://github.com/user-attachments/assets/3808323d-1ea1-40e8-a21a-5c4f7ad4aeef)

Reply:
I would prefer just purely repair action